safe_rcm/__init__.py

```diff
@@ -3,6 +3,6 @@
 from .api import open_rcm  # noqa: F401
 
 try:
-    __version__ = version("safe_rcm")
+    __version__ = version("xarray-safe-rcm")
 except Exception:
     __version__ = "999"
```

Comment:
Since we're at it: we should change the fallback version to `9999`, otherwise it will be smaller than the CalVer version we're using right now.
```suggestion
    __version__ = "9999"
```
(and the same for `tool.setuptools_scm.fallback_version` in `pyproject.toml`)